portfolio/src/main/webapp/index.html

```diff
@@ -74,6 +74,16 @@ <h2>More about me</h2>
             <p>I am not just my heritage, I am a young man with a goal to have others understand who I am and surround myself with people who will support me and challenge me. Education is what is going to help me achieve understanding, but I believe it is through the process that I will gain the relationships and knowledge that will lead to creating the network of people I strive to be a part of. That’s why I was so excited to go to college: because one of the fundamental functions of college is to bring a many of these kinds of people together. It gives me the ability to actualize my aspirations.
               </p>
             </div>
+          <div class="topic">
+            <h2>Comments</h2>
+            <div id = "comments-container"></div>
+            <form action="/data" method="POST">
+              <textarea name="name-input">Name</textarea>
```

Comment:
Should we make the `name-input` as regular text instead of `textarea`?

Reply:
Good idea, I'll change that.